packages/@okta/vuepress-site/docs/guides/refresh-tokens/get-refresh-token/index.md

```diff
@@ -34,19 +34,42 @@ GET https://${yourOktaDomain}/oauth2/default/v1/authorize?client_id={clientId}
  &state=1
 ```
 
-#### Example request for an access token, ID token, and refresh token
+The following is an example request to the `/authorize` endpoint for an [authorization code with PKCE](/docs/guides/implement-auth-code-pkce/overview/) and includes the `offline_access` scope.
 
-The following is an example request to the `/token` endpoint to obtain an access token, an ID token (by including the `openid` scope), and a refresh token. The value for `code` is the code that you receive in the response from the request to the `/authorize` endpoint.
+```bash
+https://${yourOktaDomain}/oauth2/default/v1/authorize?client_id={clientId}
+&response_type=code
+&scope=openid%20offline_access
+&redirect_uri=yourApp%3A%2Fcallback
+&state=state-8600b31f-52d1-4dca-987c-386e3d8967e9
+&code_challenge_method=S256
+&code_challenge=qjrzSW9gMiUgpUvqgEPE4_-8swvyCtfOVvg55o5S_es
+```
+
+### Example request for an access token, ID token, and refresh token
+
+The following is an example request to the `/token` endpoint to obtain an access token, an ID token (by including the `openid` scope), and a refresh token for the [Authorization Code flow](/docs/guides/implement-auth-code/overview/). The value for `code` is the code that you receive in the response from the request to the `/authorize` endpoint.
```

Comment:
The value for `code` is the code -> The value for `code` is the authorization code